inst/include/RcppArmadilloAs.h

```diff
@@ -403,14 +403,61 @@ namespace traits {
                     res = symmatl(res);
                 }
             }
+            else if (type == "pMatrix") {
+                IntegerVector i(ncol);
+                IntegerVector p(ncol+1);
```

Comment:
Sorry, what do you mean? 

For the example you provide, when converted to dgCMatrix,
`i: 0 1 2
 p: 0 1 2 3
 x:  1 1 1`